doc/container.md

```diff
@@ -222,9 +222,9 @@ archive, which helps greatly with container upgrades (see below):
 Upgrading a Container Image
 ---------------------------
 
-> **Note:** the default writable layer is lost when upgrading the image
-> Use named volumes for directories with writable content you wish to
-> keep over an upgrade.
+> **Note:** the default writable layer is lost when upgrading the image.
+> Use named volumes for directories with writable content that you wish
+> to keep over an upgrade.
```

Comment:
```suggestion
> Use named volumes for directories with writable content that you wish
> to keep over an upgrade.
```
```suggestion
> Use named volumes for content that you want to persist across upgrades.
```

Something about "keep over an upgrade" doesn't feel quite right, but this is just my opinion.